src/main/java/org/zalando/jackson/datatype/money/MoneyModule.java

```diff
@@ -64,22 +66,71 @@ public void setupModule(final SetupContext context) {
         deserializers.addDeserializer(NumberValue.class, new DecimalNumberValueDeserializer());
         deserializers.addDeserializer(MonetaryAmount.class, new MonetaryAmountDeserializer<>(amountFactory, names));
 
+        // for reading into concrete implementation types
         deserializers.addDeserializer(Money.class, new MonetaryAmountDeserializer<>(new MoneyFactory(), names));
         deserializers.addDeserializer(FastMoney.class, new MonetaryAmountDeserializer<>(new FastMoneyFactory(), names));
-        deserializers.addDeserializer(RoundedMoney.class, new MonetaryAmountDeserializer<>(new RoundedMoneyFactory(), names));
+        deserializers.addDeserializer(RoundedMoney.class, new MonetaryAmountDeserializer<>(new RoundedMoneyFactory(Monetary.getDefaultRounding()), names));
 
         context.addDeserializers(deserializers);
     }
 
-    public MoneyModule withNumberValueSerializer(final JsonSerializer<NumberValue> numberValueSerializer) {
+    public MoneyModule withDecimalNumbers() {
```

Comment:
Having it here even further reduces the public API. We only have a single public class (well two, since there is also FieldNames). Having instance methods also allows for some easier usage since the IDEs will autocomplete it.